ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/AbstractBlockPropagationManagerTest.java

```diff
@@ -815,6 +815,8 @@ public void shouldStopWhenTTDReached() {
     blockPropagationManager.start();
     syncState.setReachedTerminalDifficulty(true);
     assertThat(blockPropagationManager.isRunning()).isFalse();
+    assertThat(ethProtocolManager.ethContext().getEthMessages().messageCodesHandled())
+        .doesNotContain(1, 7);
   }
```

Comment:
maybe worth a comment to say what 1 and 7 are here, or use constants EthPV62.NEW_BLOCK